model_compression_toolkit/target_platform_capabilities/schema/v1.py

```diff
@@ -13,66 +13,74 @@
 # limitations under the License.
 # ==============================================================================
 import pprint
-
 from enum import Enum
 from typing import Dict, Any, Union, Tuple, List, Optional, Literal, Annotated
+
+from pydantic import BaseModel, Field, root_validator, validator, PositiveInt
+
 from mct_quantizers import QuantizationMethod
 from model_compression_toolkit.constants import FLOAT_BITWIDTH
 from model_compression_toolkit.logger import Logger
-from pydantic import BaseModel, Field, root_validator, validator, PositiveInt, PrivateAttr
-
-
-class OperatorSetNames(Enum):
-    OPSET_CONV = "Conv"
-    OPSET_DEPTHWISE_CONV = "DepthwiseConv2D"
-    OPSET_CONV_TRANSPOSE = "ConvTranspose"
-    OPSET_FULLY_CONNECTED = "FullyConnected"
-    OPSET_CONCATENATE = "Concatenate"
-    OPSET_STACK = "Stack"
-    OPSET_UNSTACK = "Unstack"
-    OPSET_GATHER = "Gather"
-    OPSET_EXPAND = "Expend"
-    OPSET_BATCH_NORM = "BatchNorm"
-    OPSET_RELU = "ReLU"
-    OPSET_RELU6 = "ReLU6"
-    OPSET_LEAKY_RELU = "LEAKYReLU"
-    OPSET_HARD_TANH = "HardTanh"
-    OPSET_ADD = "Add"
-    OPSET_SUB = "Sub"
-    OPSET_MUL = "Mul"
-    OPSET_DIV = "Div"
-    OPSET_MIN = "Min"
-    OPSET_MAX = "Max"
-    OPSET_PRELU = "PReLU"
-    OPSET_SWISH = "Swish"
-    OPSET_SIGMOID = "Sigmoid"
-    OPSET_TANH = "Tanh"
-    OPSET_GELU = "Gelu"
-    OPSET_HARDSIGMOID = "HardSigmoid"
-    OPSET_HARDSWISH = "HardSwish"
-    OPSET_FLATTEN = "Flatten"
-    OPSET_GET_ITEM = "GetItem"
-    OPSET_RESHAPE = "Reshape"
-    OPSET_UNSQUEEZE = "Unsqueeze"
-    OPSET_SQUEEZE = "Squeeze"
-    OPSET_PERMUTE = "Permute"
-    OPSET_TRANSPOSE = "Transpose"
-    OPSET_DROPOUT = "Dropout"
-    OPSET_SPLIT = "Split"
-    OPSET_CHUNK = "Chunk"
-    OPSET_MAXPOOL = "MaxPool"
-    OPSET_SIZE = "Size"
-    OPSET_SHAPE = "Shape"
-    OPSET_EQUAL = "Equal"
-    OPSET_ARGMAX = "ArgMax"
-    OPSET_TOPK = "TopK"
-    OPSET_FAKE_QUANT_WITH_MIN_MAX_VARS = "FakeQuantWithMinMaxVars"
-    OPSET_COMBINED_NON_MAX_SUPPRESSION = "CombinedNonMaxSuppression"
-    OPSET_CROPPING2D = "Cropping2D"
-    OPSET_ZERO_PADDING2d = "ZeroPadding2D"
-    OPSET_CAST = "Cast"
-    OPSET_STRIDED_SLICE = "StridedSlice"
-    OPSET_SSD_POST_PROCESS = "SSDPostProcess"
+
+
+class OperatorSetNames(str, Enum):
+    CONV = "Conv"
+    DEPTHWISE_CONV = "DepthwiseConv2D"
+    CONV_TRANSPOSE = "ConvTranspose"
+    FULLY_CONNECTED = "FullyConnected"
+    CONCATENATE = "Concatenate"
+    STACK = "Stack"
+    UNSTACK = "Unstack"
+    GATHER = "Gather"
+    EXPAND = "Expend"
+    BATCH_NORM = "BatchNorm"
+    L2NORM = "L2Norm"
+    RELU = "ReLU"
+    RELU6 = "ReLU6"
+    LEAKY_RELU = "LEAKYReLU"
```

Comment:
why not LeakyReLU?

Reply:
I don't know you tell me, you gave the values for this enum 😆 
I'll change it